### DISCUSSION TITLE
inputting are depth maps

### DISCUSSION THREAD

**Discussion by andy500** [nagadomi/nunif#210] [Ideas]
is it possible for you to include away of being able to input depth maps as then we can have full size depth maps of are clips and it would make the conversion faster to

**Comment by nagadomi** [comment]
iw3 already supports RGB+Depth input. See https://github.com/nagadomi/nunif/issues/97#issuecomment-2027349722

**Reply by andy500** [reply]
yes but you have to convert as out put to expect  not to be able to input your own depth maps so it can use them for conversion you can do this in the new update of Owl3D l understand now one question if l am using a 4k depth and 4k images would l need to change the yml file

**Reply by nagadomi** [reply]
If you make the same directory structure and yml file as the above export feature, you can import depth images output by other software.
For 4k depth, see https://github.com/nagadomi/nunif/discussions/180#discussioncomment-10072507 .
